storage/diemdb/src/lib.rs

```diff
@@ -764,17 +764,36 @@ impl DbReader for DiemDB {
                 ledger_info.version(),
             );
             let known_epoch = self.ledger_store.get_epoch(known_version)?;
-            let epoch_change_proof = if known_epoch < ledger_info.next_block_epoch() {
-                let (ledger_infos_with_sigs, more) = self
-                    .get_epoch_ending_ledger_infos(known_epoch, ledger_info.next_block_epoch())?;
+            let end_epoch = ledger_info.next_block_epoch();
+            let epoch_change_proof = if known_epoch < end_epoch {
+                let (ledger_infos_with_sigs, more) =
+                    self.get_epoch_ending_ledger_infos(known_epoch, end_epoch)?;
                 EpochChangeProof::new(ledger_infos_with_sigs, more)
             } else {
                 EpochChangeProof::new(vec![], /* more = */ false)
             };
 
+            // Only return a consistency proof up to the verifiable end LI. If a
+            // client still needs to sync more epoch change LI's, then they cannot
+            // verify the latest LI nor verify a consistency proof up to the latest
+            // LI. If the client needs more epochs, we just return the consistency
+            // proof up to the last epoch change LI.
+            let verifiable_li = if epoch_change_proof.more {
```

Comment:
Maybe I'm missing the context.  What's `more`?

Reply:
it's a signal in `EpochChangeProof` that the client needs to sync more epoch changes for them to fully catch up to the latest ledger state.